src/sentry/web/api.py

```diff
@@ -750,18 +751,21 @@ def post(self, request, project, **kwargs):
             if has_event_attachments:
                 attachments.append(CachedAttachment.from_upload(file))
 
-        try:
-            state = process_minidump(minidump)
-            merge_process_state_event(data, state)
-        except ProcessMinidumpError as e:
-            minidumps_logger.exception(e)
-            track_outcome(
-                project.organization_id,
-                project.id,
-                None,
-                Outcome.INVALID,
-                "process_minidump")
-            raise APIError(e.message.split('\n', 1)[0])
+        if project.id in (options.get('symbolicator.minidump-refactor-projects-opt-in') or ()):
```

Comment:
@tonyo Please confirm the use of options in the store API. Since there's only a low amount of minidumps coming in, I see no major problem with this. 

@untitaker Please also add a second check for a sampling rate, similar to `_is_symbolicator_enabled`.

Reply:
This is modelled after the option we used to sample rust store normalization, so we can be sure it's fine.

Reply:
We have random sampling, opt-in and opt-out now